tokio-postgres/src/client.rs

```diff
@@ -433,6 +435,26 @@ impl Client {
         copy_out::copy_out(self.inner(), statement).await
     }
 
+    /// Executes a 'START_REPLICATION SLOT ...', returning a stream of raw replication events
+    pub async fn start_replication(&self, query: &str) -> Result<ReplicationStream, Error> {
+        replication::start_replication(self.inner(), query).await
+    }
+
+    /// Stoppes the current replication by sending a copy_done message
+    pub async fn stop_replication(&self) -> Result<(), Error> {
+        replication::stop_replication(self.inner()).await
+    }
+
+    /// Notifies PostgreSQL of the last processed WAL
+    pub async fn standby_status_update(
```

Comment:
This is sent as part of the CopyBoth stream, not at the top level: https://www.postgresql.org/docs/13/protocol-replication.html.